Builds are failing on the run extra-tests check.

Builds are failing on the run extra-steps Action. 

Possibly related to Open issue #224 

Example build output 

`Run npm run extra-tests
npm ERR! Missing script: "extra-tests"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2022-08-01T12_15_[4](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1/runs/7610994022?check_suite_focus=true#step:4:5)[5](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1/runs/7610994022?check_suite_focus=true#step:4:6)_42[8](https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week1/runs/7610994022?check_suite_focus=true#step:4:9)Z-debug-0.log
Error: Process completed with exit code 1.`

--- a/.github/workflows/extra-tests.yml
+++ b/.github/workflows/extra-tests.yml
@@ -11,4 +11,4 @@ jobs:
     - name: Install modules
       run: npm install
     - name: Run extra tests
-      run: npm run extra-tests
+      run: npm test -- --selectProjects extra

--- a/.github/workflows/mandatory-tests.yml
+++ b/.github/workflows/mandatory-tests.yml
@@ -8,4 +8,4 @@ jobs:
     - name: Install modules
       run: npm install
     - name: Run mandatory tests
-      run: npm test
+      run: npm test -- --selectProjects mandatory

